native/src/resetprop/resetprop.cpp

```diff
@@ -118,6 +118,7 @@ static void read_prop(const prop_info *pi, void *cb) {
 struct sysprop_stub {
     virtual int setprop(const char *name, const char *value, bool trigger) { return 1; }
     virtual string getprop(const char *name, bool persist) { return string(); }
+    virtual const char* getcontext(const char *name) { return nullptr; }
```

Comment:
should return `std::string` like `getprop`

Reply:
I think using `std::string` anywhere is a very bad idea. I won't do it. Closing pull request.